SPECS/openssl/0032-Force-fips.patch

```diff
@@ -0,0 +1,82 @@
+From 97edba1a35e1cc40b8b700aef75dd8904bcb2f33 Mon Sep 17 00:00:00 2001
+From: Maxwell Moyer-McKee <mamckee@microsoft.com>
+Date: Thu, 25 Jan 2024 16:48:47 -0800
+Subject: [PATCH] Load the symcrypt provider by default if fips flag or
```

Comment:
What is the flow if the symcrypt packge is not installed, but fips is on? Out of scope for this PR but we have a validation tool for image configurations, and some documentation for FIPS, that might need to be updated if so.

Reply:
I suppose it won't be without precedent... the system won't even boot without `dracut-fips` installed if you turn on fips mode....

Can we add this package to `imageconfigvalidator.go:validatePackages()` so we at least yell about it missing if fips is configured in the image json?

@tobiasb-ms We will want to update compliance and documentation to reflect this, or do we want to add a Requires on symcrpt?

Reply:
Had a chat offline, we can just ignore this for now and fix it later in the 3.0 release. Figure out what we want to do.